.github/workflows/ci.yml

```diff
@@ -310,3 +310,29 @@ jobs:
       os: ubuntu-latest
       node-version: 18
     secrets: inherit
+
+  check-all-jobs:
+    if: always()
+
+    needs:
+    - lint-styles
+    - lint-js
+    - build-ubuntu-node-16
+    - build-windows-node-16
+    - build-ubuntu-node-18
+    - build-windows-node-18
+    - unit-tests
+    - browser-tests-components
+    - deploy-docs
```

Comment:
will this be ignored if the PR doesn't require a docs/storybook deployment?

Reply:
That's the idea Jamie. See line 337.

Reply:
perfect 👍